client/store/usersStore.js

```diff
@@ -30,12 +31,13 @@ const createUserActionCreator = (user) => {
 //Create User Thunk
 const createUser = (email, password, firstName, lastName, history) => {
```

Comment:
Another way of doing this is that you can just pass all this info as an object instead of each seperate field. So like (newUser, history) and same in the axios call and just destructor it in your route call in the backend.

Reply:
Thank you, ya, I can see how that would be cleaner. Is it cool if I stick with how it is right now, and come back to it for some refactoring later once I have the login page more complete?

Reply:
Yeah of course! Not a biggie